microsoft/botframework-solutions
# [Enterprise Bot][C#] Clean up sample and template

## Description

Enterprise Bot Sample
- Delete unused `EnterpriseBotSampleState.cs`.
- Delete unused `using` statements in several files.
- Add _Copyright header_ to all files.

Enterprise Bot Template
- Replicate changes from Sample
- Delete autogenerated `.zip` file.

## Related Issue




#524 **[Enterprise Bot Template][C#]** The sample contains unused files and classes with unused dependencies. 

## Testing Steps


1. Navigate to `templates\Enterprise-Template\src\csharp\`
1. Open `EnterpriseTemplate.sln`
1. Build and Run the solution.
1. Create a new Project and select `EnterpriseBot` as template.
1. Verify that all files have propper dependencies and _Copyright_ header.

## Checklist
